cmd/br-federation-manager/main.go

```diff
@@ -176,60 +175,17 @@ func main() {
 	}
 	// leader election for multiple br-federation-manager instances
 	go wait.Forever(func() {
-		var lock resourcelock.Interface
-		switch cliCfg.ResourceLock {
-		case resourcelock.EndpointsResourceLock:
-			lock = &resourcelock.EndpointsLock{
```

Comment:
`resourcelock.EndpointsLock` can not be accessed now, ref https://github.com/kubernetes/kubernetes/pull/106852/files#diff-4c70f2593ad888d66837263e010a1028d63317aeeb56e6b2598826f63f2b4775R30

Reply:
Why not just print fatal error if the `ResourceLock` is `EndpointsResourceLock`? Does the `resourcelock.New()` method check the lock type?

Reply:
`EndpointsResourceLock` will get a `endpoints lock is removed, migrate to endpointsleases` and then, we'll `klog.Fatalf("failed to create lock: %v", err)`.